app/admin/travels.rb

```diff
@@ -19,6 +18,20 @@
   #   permitted
   # end
 
+  filter :enrollment_applicant_detail_lastname_start, label: "Last Name (Starts with)"
+  filter :enrollment_applicant_detail_firstname_start, label: "First Name (Starts with)"
+  filter :arrival_session, label: "Session of Arrival", as: :select, collection: CampOccurrence.no_any_session.map { |s| s.description_with_month_and_day }
+  filter :depart_session, label: "Session of Departure", as: :select, collection: CampOccurrence.no_any_session.map { |s| s.description_with_month_and_day }
+  filter :arrival_date
+  filter :depart_date, label: "Departure Date"
+  filter :arrival_transport, as: :select, collection: ['Airplane', 'Bus', 'Train', 
```

Comment:
If you use 
`filter :arrival_transport, as: :select`

Then the list will populate only the values used in any of the records. Let's go that route to DRY this up